test/functional/core/job_spec.lua

```diff
@@ -1,3 +1,4 @@
+local global_helpers = require('test.helpers')
```

Comment:
never needed, because test.helpers is already merged-into functional.test.helpers:

https://github.com/neovim/neovim/blob/ac73f75fc0073db3e0bb1c0c2758b3dc34a3ed51/test/functional/helpers.lua#L869